tgui/src/interfaces/bsa.ract

```diff
@@ -9,7 +9,13 @@
     <ui-button icon='crosshairs' action='recalibrate'>{{data.target}}</ui-button>
   </ui-section>
   <ui-section label='Controls'>
-    <ui-button icon='warning' state='{{data.ready ? null : "disabled"}}' action='fire'>FIRE!</ui-button>
+	{{#if !data.unlocked}}
+		<ui-notice>
+			<span>Bluespace Artillery firing protocals must be globally unlocked from two keycard authentication devices first!</span>
```

Comment:
I don't like the `must be globallly unlocked` part, doesn't mean much in the game context.